### PR TITLE
[Homework #6] - JS Basic 1 by Stanisław Królikiewicz

### PR DESCRIPTION
**Pull request by stkrolikiewicz**

### CODE REVIEW

File: homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+      return value1 * value2
+    case '/':
+      return value1 / value2
+    default:
+      break
+  }
+}
+
+// task 3 http://www.codewars.com/kata/printing-array-elements-with-comma-delimiters
+const printArray = (Array) => {
```
Comment (solvd-mmelnyk) [inline homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js:21]: please do not use uppercase first letter for variable name

File: homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+const times = (rightOperand) => {
+  return '*' + rightOperand
+}
+const dividedBy = (rightOperand) => {
+  return '/' + rightOperand
+}
+
+// task 6 http://www.codewars.com/kata/get-the-middle-character
+const getMiddle = (word) => {
+  let middle = parseInt(word.length/2)
```
Comment (solvd-mmelnyk) [inline homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js:140]: please use const if you don't need to change variable

File: homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+  arr.forEach(item => pred(item) ? tsArray.push(item) : fsArray.push(item))
+  arr.splice(0, arr.length, ...fsArray.concat(tsArray))
+  return tsArray.length > 0 ? fsArray.length : 'There is no boundary index! All the values are false.'
+}
+
+// task 8 http://www.codewars.com/kata/word-count
+// Error 404
+
+// task 9 https://www.codewars.com/kata/find-the-odd-int/
+const findOdd = (Array) => {
```
Comment (solvd-mmelnyk) [inline homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js:157]: do not use uppercase for first word letter

File: homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+      lastLastN = lastN
+      lastN = sum
+    }
+  }
+  return sum
+}
+
+// task 14 https://www.codewars.com/kata/cat-and-mouse-2d-version/
+const catMouse = (map, moves) => {
+  let C = {row: undefined, col: undefined}, m = {row: undefined, col: undefined}, minMoves, width = 0
```
Comment (solvd-mmelnyk) [inline homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js:226]: please use more meaningful variable name
it is hard to understand what this means

**Review by solvd-mmelnyk — CHANGES_REQUESTED** [review]
Please make sure that you format your code with prettier!
please take a look at my comments and make sure that you are using the correct variable name and meaningful variable names

File: homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js (hunk 4 above)
Comment (stkrolikiewicz) [reply homeworks/stanislaw-krolikiewicz/06-js-basics-1/script.js]: Ok, I've provided changes. Let me know if it is ok.

**Comment by stkrolikiewicz** [comment]
I've provided the changes.

**Review by solvd-mmelnyk — APPROVED** [review]

**Pull request merged**